frontend/src/screens/internal-apps/ZapPlanner.tsx

```diff
@@ -141,14 +224,17 @@ export function ZapPlanner() {
             "Content-Type": "application/json",
           },
           body: JSON.stringify({
-            recipientLightningAddress: recipientLightningAddress,
-            amount: parsedAmount,
+            recipientLightningAddress,
+            amount: satsToSend, // ← always sats
```

Comment:
this is not correct, for fiat you provide the fiat value and fiat currency. ZapPlanner will calculate the sat value for every single payment, therefore the payment amount will be "stable" in fiat terms (which is the whole point of adding this feature)